tests/test_s3.py

```diff
@@ -225,6 +231,17 @@ def test_threading_false(self):
 
             self.test_local()
 
+    @unittest.skipIf(fsspec is None, "fsspec is not available")
+    def test_fsspec_constructor(self) -> None:
+        """Test that we can obtain an s3fs object."""
+        uri = self.root_uri.join("test_file.dat")
+        fs, path = uri.to_fsspec()
+        self.assertEqual(path, f"{uri._bucket}/{uri.relativeToPathRoot}")
+        self.assertTrue(hasattr(fs, "open"))
+
+    def test_fsspec(self) -> None:
+        raise unittest.SkipTest("fsspec s3fs incompatible with moto")
```

Comment:
Using moto in [server mode](https://docs.getmoto.org/en/stable/docs/server_mode.html#start-within-python) would be one way to get this test working.  Might not be worth the trouble though.

Reply:
Yes. I found a couple of posts online about changing to server mode and decided that was more work reorganizing all the tests than seemed warranted for the moment.